iox_query/src/exec/gapfill/params.rs

```diff
@@ -211,21 +227,58 @@ mod tests {
                \ngroup by minute",
             ).await?;
         let expected = GapFillParams {
-            stride: 60_000_000_000,            // 1 minute
-            first_ts: 441_820_449_000_000_000, // Sunday, January 1, 1984 3:54:09 PM
-            last_ts: 441_820_749_000_000_000,  // Sunday, January 1, 1984 3:59:09 PM
+            stride: 60_000_000_000,                  // 1 minute
+            first_ts: Some(441_820_449_000_000_000), // Sunday, January 1, 1984 3:54:09 PM
+            last_ts: 441_820_749_000_000_000,        // Sunday, January 1, 1984 3:59:09 PM
         };
         assert_eq!(expected, actual);
         Ok(())
     }
 
+    fn interval(ns: i64) -> Arc<dyn PhysicalExpr> {
+        Arc::new(Literal::new(ScalarValue::IntervalDayTime(Some(
+            ns / 1_000_000,
+        ))))
+    }
+
+    fn timestamp(ns: i64) -> Arc<dyn PhysicalExpr> {
+        Arc::new(Literal::new(ScalarValue::TimestampNanosecond(
+            Some(ns),
+            None,
+        )))
+    }
+
+    #[test]
+    fn test_params_no_start() -> Result<()> {
+        let exec_params = GapFillExecParams {
+            stride: interval(1_000_000_000),
+            time_column: Column::new("time", 0),
+            origin: timestamp(0),
+            time_range: Range {
+                start: Bound::Unbounded,
+                end: Bound::Excluded(timestamp(20_000_000_000)),
+            },
+        };
+
+        let actual = GapFillParams::try_new(schema().into(), &exec_params)?;
```

Comment:
Not a requirement, just an alternative. 

```suggestion
        let actual = GapFillParams::try_new(schema().into(), &exec_params).unwrap();
```

You can `unwrap()` the result, which avoids having to add `-> Result<()>` and the `Ok(())` at the end of the test. A panic will occur on the line of the `unwrap`, if that condition fails, which is helpful for debugging. This tip was suggested to me by Marco.

Reply:
I like it, I'll start writing tests this way.